scripts/start_server.sh

```diff
@@ -1,17 +1,10 @@
 #!/usr/bin/env bash
```

Comment:
If this script is intended to be used outside of docker as well, then I would suggest editing the makefile to run the script instead of having it the other way around since `ensure_connected_services` has been removed.

Reply:
Hmm not too sure on this anymore since it relies on wait4x

Reply:
on that topic - have any opinion on [wait4x](https://github.com/atkrad/wait4x) vs. [wait-for-it](https://github.com/vishnubob/wait-for-it)?
i've only used wait-for-it, it has ~20x more stars (so i assume more prod tested)

Reply:
I would prefer wait-for-it myself for two main reasons:
1. wait4x seems to be fairly heavily focused on alpine linux (something which we don't want)
2. wait-for-it is a simple bash script which could easily be included (would solve the concern I had)

Reply:
I saw that wait4x was in the [alpine repo](https://pkgs.alpinelinux.org/packages?name=wait4x&branch=edge&repo=&arch=&maintainer=) and decided to use it, but if we're going to switch to slim then let's use wait-for-it

Reply:
wait-for-it is licensed under the MIT license, therefore distributing it ourselves is an option. @cmyui do you have any thoughts on this?

Reply:
Yeah I've typically done that, [or have written by own lightweight version of it](https://github.com/akatsuki-v2/users-service/blob/main/scripts/await-service.sh) for my own usage.

Perhaps there's an argument to use something like a submodule to keep it up to date automatically, but I'll defer that choice to whoever implements it (seems rather minor imo).